app/src/main/java/com/example/sharingang/EditItemFragment.kt

```diff
@@ -17,34 +17,45 @@ class EditItemFragment : Fragment() {
     private val viewModel: ItemsViewModel by activityViewModels()
     private lateinit var existingItem: Item
 
+    private lateinit var binding: FragmentEditItemBinding
+
     override fun onCreateView(
-            inflater: LayoutInflater, container: ViewGroup?,
-            savedInstanceState: Bundle?
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?

```

Comment:
This formatting looks odd to me, has this file been formatted? I'd expect to see the items on multiple lines, if the last item has been moved to a new line

Reply:
It is formatter as well